manifests/s/Samsung/DataMigration/4.0.0.18/Samsung.DataMigration.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.DataMigration
+PackageVersion: 4.0.0.18
+PackageLocale: en-US
+Publisher: Samsung
+PackageName: Samsung Data Migration
+License: Proprietary
+Copyright: Copyright (c) 2015 Flexera Software LLC. All Rights Reserved.
+ShortDescription: Samsung Data Migration

```

Comment:
Please provide a better short description of what `Samsung Data Migration` does / is used for